Sources/KlaviyoSwift/Klaviyo.swift

```diff
@@ -13,6 +13,115 @@ let version = "2.0.0"
 
 public class Klaviyo: NSObject  {
     
+    public struct Event {
+        public struct Attributes {
+            public struct Metric {
```

Comment:
can we split this into its own file?

Reply:
Uhh this is gonna be public API - I was thinking it might be easier to read api docs with this nesting.

Reply:
Punting on this made this whole in a separate file - should solve both problems.